gleam/tmist/gleam.toml

```diff
@@ -0,0 +1,23 @@
+name = "tmist"
+version = "1.0.0"
+
+# Fill out these fields if you intend to generate HTML documentation or publish
+# your project to the Hex package manager.
+#
+# description = ""
+# licences = ["Apache-2.0"]
+# repository = { type = "github", user = "", repo = "" }
+# links = [{ title = "Website", href = "" }]
+#
+# For a full reference of all the available options, you can have a look at
+# https://gleam.run/writing-gleam/gleam-toml/.
+
+[dependencies]
+gleam_stdlib = ">= 0.34.0 and < 2.0.0"
+mist = ">= 3.0.0 and < 4.0.0"
+gleam_erlang = ">= 0.32.0 and < 1.0.0"
+gleam_http = ">= 3.7.1 and < 4.0.0"
+gleam_otp = ">= 0.14.1 and < 1.0.0"
```

Comment:
not needed -> https://github.com/rawhat/mist/blob/d00d66e1e28bc72e8a47a6267684cced0411e180/gleam.toml#L17